Example/Core/Tests/FIRAppTest.m

```diff
@@ -715,6 +716,28 @@ - (void)testGlobalDataCollectionNoDiagnosticsSent {
   OCMVerifyAll(self.observerMock);
 }
 
+- (void)testGlobalDataCollectionNoDiagnosticsSentForNoOptions {
+  // Add an observer for the diagnostics notification - both with and without an object to ensure it
+  // catches it either way. Currently no object is sent, but in the future that could change.
+  [self.notificationCenter addMockObserver:self.observerMock
+                                      name:kFIRAppDiagnosticsNotification
+                                    object:nil];
+  [self.notificationCenter addMockObserver:self.observerMock
+                                      name:kFIRAppDiagnosticsNotification
+                                    object:OCMOCK_ANY];
```

Comment:
Does OCMOCK_ANY include nil?

Reply:
Will test to see if that's the case!

Reply:
Good call,  it does include nil. I'll make sure we don't duplicate this elsewhere in a different PR (grabbed this from a different location).

Reply:
Done.